### PR TITLE
adding dicom-viewer.yml

### PR DESCRIPTION
**Pull request by koebel**
## Description
enables admins to add DICOM Viewer with only a few lines in .env to the oCIS deployment example

## Related Issue

- relates to https://github.com/owncloud/web-app-dicom-viewer/issues/64

## Motivation and Context

make it as easy as possible for admins to add the extension to their installation

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added

### CODE REVIEW

File: dicom-viewer.yml
Comment (saw-jan) [inline dicom-viewer.yml:1]: how can I use this? is there any docs?

Comment (koebel) [reply dicom-viewer.yml:1]: see https://github.com/owncloud/web-app-dicom-viewer/issues/64, section adding Dicom Viewer to ocis Deployment Example

Comment (saw-jan) [reply dicom-viewer.yml:1]: got it.

**Review by saw-jan — APPROVED** [review]
looks good :+1:

File: dicom-viewer.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+services:
+  ocis:
+    depends_on:
+      dicomviewer-init:
+        condition: service_completed_successfully
+
+  dicomviewer-init:
+    image: owncloud/web-app-dicom-viewer/dicom-viewer-1.0.0
```
Comment (koebel) [inline dicom-viewer.yml:9]: we still have to create the corresponding docker image!

Comment (koebel) [reply dicom-viewer.yml:9]: @S-Panta this is the path to the docker image, ensure that it is correct after creating the actual docker image on  docker hub. If we follow the same structure/naming conventions like other ownCloud apps seem to use, it should be https://hub.docker.com/u/owncloud/web-app-dicom-viewer and for the tag dicom-viewer-1.0.0

File: dicom-viewer.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    user: root
+    volumes:
+      - ocis-apps:/apps
+    entrypoint:
+      - /bin/sh
+    command: ["-c", "cp -R /var/lib/nginx/html/dicom-viewer/ /apps"]
```
Comment (S-Panta) [inline dicom-viewer.yml:15]:
```suggestion
    command: ["-c", "cp -R /app/js/* /apps"]
```
I haven't used nginx but simple node alpine to create image. I don't think this make any differences

Comment (saw-jan) [reply dicom-viewer.yml:15]: path depends on how it is done in the docker image

File: dicom-viewer.yml (hunk 1 above)
Comment (S-Panta) [reply dicom-viewer.yml:9]: The name should be `owncloud/web-app-dicom-viewer:latest` for now
https://hub.docker.com/r/owncloud/web-app-dicom-viewer/tags

Comment (koebel) [inline dicom-viewer.yml:9]:
```suggestion
    image: owncloud/web-app-dicom-viewer:latest
```

**Review by S-Panta — APPROVED** [review]
LGTM

File: dicom-viewer.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        condition: service_completed_successfully
+
+  dicomviewer-init:
+    image: owncloud/web-app-dicom-viewer:latest
+    user: root
+    volumes:
+      - ocis-apps:/apps
+    entrypoint:
+      - /bin/sh
+    command: ["-c", "cp -R /var/lib/nginx/html/dicom-viewer/ /apps"]
```
Comment (saw-jan) [inline dicom-viewer.yml:15]: Is this the correct path?

**Review by saw-jan — CHANGES_REQUESTED** [review]

File: dicom-viewer.yml (hunk 3 above)
Comment (S-Panta) [reply dicom-viewer.yml:15]: My bad. the path is 
```
command: ["-c", "cp -R /app /apps/dicom-viewer"]
```

**Pull request merged**